org.eclipse.nebula.widgets.nattable.core.feature/feature.xml

```diff
@@ -19,18 +19,13 @@
       %license
    </license>
 
-   <plugin
-         id="org.eclipse.nebula.widgets.nattable.core"
-         download-size="0"
-         install-size="0"
-         version="0.0.0"
-         unpack="false"/>
+   <requires>
+      <import plugin="org.eclipse.collections.api"/>
+      <import plugin="org.eclipse.collections.impl"/>
```

Comment:
```suggestion
      <import plugin="org.eclipse.collections.impl"/>
```

I assume the API is already referenced somehow and does not needs to be listed explicitly?

Reply:
Typically the impl bundle is not referenced directly but only the api bundle. In the older versions of Eclipse Collections the factory.primitive bundle was not available in the api bundle, which is why there is such a dependency to impl in NatTable. But it seems that was fixed with 11.1.0 already. So I will change that now too.

Reply:
They have pretty extensive capability headers.

```
Require-Capability: osgi.extender;filter:="(&(osgi.extender=osgi.service
 loader.processor)(version>=1.0.0)(!(version>=2.0.0)))",osgi.serviceload
 er;filter:="(osgi.serviceloader=org.eclipse.collections.api.factory.bag
 .ImmutableBagFactory)";...
```

```
Provide-Capability: osgi.service;objectClass:List<String>="org.eclipse.c
 ollections.api.factory.bag.ImmutableBagFactory";effective:=active,...
```

That was actually my main contribution to Eclipse Collections. Using bndtools to generate the manifest files for correct SPI headers.

Does that mean I can also remove the IMPL dependency?

Reply:
It seems to be the case ...